packages/cli/src/tools/config/index.js

```diff
@@ -22,8 +22,7 @@ import assign from '../assign';
 /**
  * Built-in platforms
  */
-import * as ios from '../ios';
-import * as android from '../android';
+import {ios, android} from '../../..';
```

Comment:
@todo let's remove it when we are done, update tests and then, ship a PR to React Native to turn this on.